binding/binding.go

```diff
@@ -43,17 +43,17 @@ type BindingUri interface {
 	BindUri(map[string][]string, interface{}) error
 }
 
-// StructValidator is the minimal interface which needs to be implemented in
+// Validater is the minimal interface which needs to be implemented in
```

Comment:
why update it? BREAK-BACKWARD, please revert.

Reply:
@thinkerou but the interface's mean's has changed,it is not suitable to using the old interface name for supporting slice validation.

Reply:
@appleboy do you think? and you should use `Validator` not `Validater`.

Reply:
> @appleboy do you think? and you should use `Validator` not `Validater`.

it will be conflict with package variable: `var Validator Validater = &defaultValidator{}`. Any suggestion? @thinkerou @appleboy